version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.1.1-SNAPSHOT"
+version in ThisBuild := "2.2.0-SNAPSHOT"
```

Comment:
Don't update the version, it is automatically updated when I do a release with sbt-release.